Dockerfile

```diff
@@ -1,31 +1,29 @@
-FROM debian:jessie
-MAINTAINER Luke Campbell <luke.campbell@rpsgroup.com>
+FROM node:13.7.0-alpine
+MAINTAINER RPS Dev-Ops <devops@rpsgroup.com>
 
-ENV NODE_VERSION 7.9.0
-ENV GOSU_VERSION 1.9
-ENV SCRIPTS_DIR /opt/build_scripts
-
-RUN mkdir -p $SCRIPTS_DIR
-RUN useradd -m node
-
-COPY contrib/scripts/ $SCRIPTS_DIR/
-
-RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
-
-RUN $SCRIPTS_DIR/install-deps.sh
-RUN $SCRIPTS_DIR/install-node.sh
 COPY bin /opt/ioos-us/bin
 COPY public /opt/ioos-us/public
 COPY routes /opt/ioos-us/routes
 COPY views /opt/ioos-us/views
-COPY .bowerrc app.js assets.json bower.json gruntfile.js package.json /opt/ioos-us/
+COPY app.js assets.json gruntfile.js package.json /opt/ioos-us/
 
+# install yarn
+RUN apk update && \
+    apk add yarn
```

Comment:
Consolidate all package management operations into one or several `RUN` instructions and make sure you are clearing out build cache within the same `RUN` instruction so that layers don't get bloated.  It's been a while since I've used Alpine, but I believe you want to remove apk cache like so: `rm -rf /var/cache/apk/*`